packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1109,7 +1136,7 @@ contract Validators is
    * @return Whether a particular address is a registered validator.
    */
   function isValidator(address account) public view returns (bool) {
-    return validators[account].publicKeys.bls.length > 0;
+    return validators[account].publicKeys.ecdsa.length > 0;
```

Comment:
shouldn't this behaviour change after the L2?

Reply:
What do you mean?

Reply:
the check should work as usual before the L1 (when this code is deployed) and then change logic. Am I right?

Reply:
This check will work in both L1 and L2 - in either case it depends only on checking some value that is non-0 for a registered validator. In fact, in prior versions of this code, it used to check a different part of the struct. So there should be no need to increase complexity with an L2 check. The check is different, but behavior is equivalent.